js/join.js

```diff
@@ -173,21 +175,39 @@ function whyRdsPageValidator() {
   return (
     whyRds.value.trim().split(' ').length >= sizeDef.whyRds &&
     foundFrom.value != '' &&
-    whyRds.value.trim() != ''
+    whyRds.value.trim() != '' &&
+    numberOfHours.value != '' &&
+    parseInt(numberOfHours.value) > 1 &&
+    parseInt(numberOfHours.value) <= 100
   );
 }
 
 function dataValidator(element, size) {
```

Comment:
Change the function name. 
E.g - `textFieldValidator`, `textValidation` etc

Reply:
ok, i will do that.